Makefile

```diff
@@ -58,3 +58,13 @@ run: wasm
 .PHONY: docker-build
 docker-build:
 	docker build -f ./docker/Dockerfile . --tag jwt-server:$(IMAGE_TAG)
+
+# Kustomize targets
```

Comment:
question: does it make sense just create a normal deploy (default serveType: clusterIP)?
```
deploy: 
   kustomize build deployments | kubectl apply -f -
```
this is probably the most comment scenario when people just develop something and experience with it.